src/main/java/org/truffleruby/core/string/StringNodes.java

```diff
@@ -1826,23 +1982,24 @@ public DynamicObject squeezeBangEmptyString(DynamicObject string, Object[] args)
         @Specialization(guards = { "!isEmpty(string)", "zeroArgs(args)" })
         @TruffleBoundary
         public Object squeezeBangZeroArgs(DynamicObject string, Object[] args,
-                                          @Cached("createBinaryProfile()") ConditionProfile singleByteOptimizableProfile) {
+                @Cached("createBinaryProfile()") ConditionProfile singleByteOptimizableProfile) {
```

Comment:
I think we line up a lot of the arguments the previous way. Replacing it seems like a good way to get into a format battle.